TMS.Net07.Homework.HelloWorld/TMS.Net07.Homework.HelloWorld/Hard.cs

```diff
@@ -19,15 +23,15 @@ static void Main ()
             if (inputIsNumber == true)
             {
                 if (number == 0)
-                    Console.WriteLine($"You are enter number {number}!\n Entering number 0 is close this program.\n Good bye!");
+                    Console.WriteLine($"{txtMessage} number {number}!\n Entering number 0 is close this program.\n Good bye!");
```

Comment:
Suggest using `Environment.NewLine` constant instead of `\n` - some systems use `\r\n` and don't understand `\n`. Value of `Environment.NewLine` depends of platform

Reply:
Done at all home tasks.